scheduler/src/cook/components.clj

```diff
@@ -334,6 +338,27 @@
                           :or {user-limit-per-m 600}} rate-limit]
                      {:user-limit (->UserRateLimit :user-limit user-limit-per-m (t/minutes 1))}))
      :sim-agent-path (fnk [] "/usr/bin/sim-agent")
+     :executor (fnk [[:config {executor {}}]]
+                 (if (seq executor)
+                   (do
+                     (when (str/blank? (:command executor))
+                       (throw (ex-info "Executor command is missing!" {:executor executor})))
+                     (when (and (:uri executor) (nil? (get-in executor [:uri :value])))
+                       (throw (ex-info "Executor uri value is missing!" {:executor executor})))
+                     (let [default-executor-config {:log-level "INFO"
+                                                    :max-message-length 512
+                                                    :progress-output-file "stdout"
```

Comment:
Consider naming this `default-progress-output-file` and `default-progress-regex-string` to signal we intend to allow users to configure this in the future.